components/brave_extension/extension/brave_extension/types/other/settingsTypes.ts

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type Settings = {
+  key: string
+  type: any
+  value: boolean
```

Comment:
nit: this won't always be a boolean but for now it's ok. Maybe leave a (`TODO: can support multiple types, see PrefType in chromel.d.ts`).

Reply:
Done